tests/preprocessing/test_block_transformer.py

```diff
@@ -0,0 +1,67 @@
+import dask
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pytest
+from dask.array.utils import assert_eq as assert_eq_ar
```

Comment:
Rather than aliasing, can you just use `da.utils.assert_eq` and `dd.utils.assert_eq`?